### DISCUSSION TITLE
Looking for Docker files to deploy pimcore 10.x to my organization

### DISCUSSION THREAD

**Discussion by san4ever** [pimcore/pimcore#9959] [Q&A / Support]
We  are planning to deploy pimcore 10.x to EKS. Can some one provide with the steps on the same

**Comment by cmann90** [comment]
You can find docker installation instructions in the pimcore skeleton repository:
https://github.com/pimcore/skeleton

Pimcore docker images:
https://github.com/pimcore/docker
https://hub.docker.com/r/pimcore/pimcore

I have no knowledge about EKS. So I cannot help there.
